src/app/containers/Ad/Amp/index.jsx

```diff
@@ -135,24 +135,16 @@ const AmpAd = () => {
             >
               {label}
             </StyledLink>
-            <amp-ad {...ampAdPropsMobile({ service })}>
-              <amp-img
-                placeholder
-                width="320"
-                height="50"
-                src="https://via.placeholder.com/320x50"
-                layout="responsive"
-              />
-            </amp-ad>
-            <amp-ad {...ampAdPropsDesktop({ service })}>
-              <amp-img
-                placeholder
-                width="970"
-                height="250"
-                src="https://via.placeholder.com/970x250"
-                layout="responsive"
-              />
-            </amp-ad>
+            <amp-ad
+              width="320"
```

Comment:
I think the width and height should already be coming from the `ampAdPropsMobile` and `ampAdPropsMobile` that are being spread here!